### ISSUE TITLE
Latest releases are not present in maven central repository 

### ISSUE THREAD

**Issue by metyl** [jwt-scala/jwt-scala#642]
Tried to download latest release `10.0.3` and it fails with:
```
[error] Error downloading com.github.jwt-scala:jwt-core_2.13:10.0.3
[error]   Not found
[error]   Not found
...
[error]   not found: https://repo1.maven.org/maven2/com/github/jwt-scala/jwt-core_2.13/10.0.3/jwt-core_2.13-10.0.3.pom
```
quick check in repo1 confirms it:
<img width="631" alt="image" src="https://github.com/user-attachments/assets/81750292-9cb3-483a-8410-061736e86fda" />

the same story for all modules(Circe, ZIO JSON, etc) and Scala versions(2.12, 2.13, 3), can you fix it?

**Comment by erwan** [comment]
Yes I have an issue with the automatic release from Github, I need to fix that

**Comment by metyl** [comment]
@erwan any forasable date for the fix?

**Comment by mijicd** [comment]
@erwan can we help fixing it? There was an attempt in #662

**Comment by erwan** [comment]
@mijicd yes, if you find the reason why publishing stopped working that would help.

I'm trying to revert the latest updates of sbt-pgp and sbt-ci-release to see if that help.

**Comment by plokhotnyuk** [comment]
Could it be that 401 error is happening due to changes on the server side?

As an example publishing with username/password could be not supported anymore and need to provide some API token instead

**Comment by erwan** [comment]
@plokhotnyuk yes it could be, I'll take a look at that

**Comment by erwan** [comment]
looking good so far for a 10.0.4 release 🤞 

https://github.com/jwt-scala/jwt-scala/actions/runs/13158160290/job/36719982806

(and indeed the issue was that username/password is no longer allowed to publish on sonatype)

**Comment by erwan** [comment]
@mijicd there can be some delay but feel free to close this if you confirm that 10.0.4 is published and usable!

**Comment by mijicd** [comment]
@erwan I can't close the issue as I'm not its author, but I can confirm that 10.0.4 is available! Thank you!
